gorapass/models.py

```diff
@@ -41,4 +43,28 @@ class Hikes(models.Model):
     completed_at_date = models.DateField()
 
     def __str__(self):
-      return self.hike_name
\ No newline at end of file
+      return self.hike_name
+
+class CompletedStamps(models.Model):
+    stamp = models.ForeignKey(Stamps, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    class Meta:
```

Comment:
I haven't heard of the Meta class before and had to look it up. Interesting way that Django makes it easier to specify the behavior of the model!